provisioning/baremetal_pod.go

```diff
@@ -584,6 +585,17 @@ func newMetal3PodTemplateSpec(images *Images, config *metal3iov1alpha1.Provision
 		},
 	}
 
+	for _, cont := range containers {
+		// these are "default" values to pass openshift e2e tests.
```

Comment:
This sounds a little bit too generic (which tests?). I guess it's better to mention the underlying requirement, ie enforcing the OpenShift convention on resources and limits for control plane components (https://github.com/openshift/enhancements/blob/master/CONVENTIONS.md#resources-and-limits)

Reply:
Yes, please don't set "default" values.  You need to set "correct" and "accurate" values ;)